packages/yew/src/scheduler.rs

```diff
@@ -122,6 +122,40 @@ pub(crate) fn start() {
     });
 }
 
+#[cfg(target_arch = "wasm32")]
+mod target_wasm {
+    use super::*;
+    use crate::io_coop::spawn_local;
+
+    /// We delay the start of the scheduler to the end of the browser event loop.
+    /// So any messages that needs to be queued can be queued.
+    pub(crate) fn start() {
+        spawn_local(async {
```

Comment:
This should put it in the micro-task queue, which isn't executed at *the end*, the task queue is. Should we be using `setTimeout(callback, 0)` or perhaps `requestsAnimationFrame` instead?

Reply:
If I understand this correctly, this will spawn multiple micro-tasks if `scheduler::start()` is called multiple times (e.g. whenever a `Scope::new` is created. Are you sure this is intended?

Reply:
It does not really matter here.
As in the future it is supposed to yield before rendering (`sleep(Duration::ZERO).await`)
and we kind of need a future here.

Reply:
> Are you sure this is intended?

It's not more expensive to check whether the scheduler is running in the task.